open-metadata-implementation/view-services/subject-area-view/subject-area-view-spring/pom.xml

```diff
@@ -7,42 +7,37 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>ui-admin-services</artifactId>
+        <artifactId>subject-area-view</artifactId>
         <groupId>org.odpi.egeria</groupId>
         <version>1.5-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <scm>
-        <connection>scm:git:git://github.com/odpi/egeria.git</connection>
-        <developerConnection>scm:git:ssh://github.com/odpi/egeria.git</developerConnection>
-        <url>http://github.com/odpi/egeria/tree/master</url>
-    </scm>
-
-    <artifactId>ui-admin-services-client</artifactId>
-
-    <name>User Interface Administration Services Client</name>
-    <description>
-        Provides a Java client for the open metadata ui administration services.
-    </description>
+    <artifactId>subject-area-view-spring</artifactId>
 
     <dependencies>
 
         <dependency>
             <groupId>org.odpi.egeria</groupId>
-            <artifactId>ui-admin-services-api</artifactId>
+            <artifactId>subject-area-view-server</artifactId>
         </dependency>
 
         <dependency>
             <groupId>org.odpi.egeria</groupId>
-            <artifactId>admin-services-api</artifactId>
+            <artifactId>subject-area-api</artifactId>
         </dependency>
 
         <dependency>
-            <groupId>org.odpi.egeria</groupId>
-            <artifactId>ffdc-services</artifactId>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.tomcat.embed</groupId>
```

Comment:
I think we might be able to do without this. You just need the servlet API

See https://github.com/odpi/egeria/issues/2267

Also consider that since the servlet implementation is PROVIDED by spring boot, it may be possible to express the dependency as <provided> which just means 'it will be there' .. ;-)

Reply:
Updated #2267 to ensure that the view services are brought into line with what ever the outcome is of that issue. For now I will leave it consistent with the existing access services